README.md

```diff
@@ -263,13 +307,13 @@ Finally, the application can be run as:
         <ssl enabled=true|false> \
         <ckan API key> \
         <comma-separated list of ckan inputs> \
-        <hdfs output folder> \
+        <ckan output package> \
         <splits length>
         
 An example of run:
 
 ```
-$ hadoop jar /home/user1/fiware-tidoop/target/tidoop-0.0.0_SNAPSHOT-jar-with-dependencies.jar com.telefonica.iot.tidoop.utils.CKANMapReduceExample -libjars /home/user1/fiware-tidoop/target/tidoop-0.0.0_SNAPSHOT-jar-with-dependencies.jar -D mapreduce.job.queuename=user1q data.lab.fiware.org 443 true 2d5bf021-ff9f-48e3-bb97-395b77581665 https://data.lab.fiware.org/dataset/logrono_cygnus/resource/ca73a799-9c71-4618-806e-7bd0ca1911f4 /user/user1/ckan_output 1000
+$ hadoop jar /home/user1/fiware-tidoop/target/tidoop-0.0.0_SNAPSHOT-jar-with-dependencies.jar com.telefonica.iot.tidoop.utils.CKANMapReduceExample -libjars /home/user1/fiware-tidoop/target/tidoop-0.0.0_SNAPSHOT-jar-with-dependencies.jar data.lab.fiware.org 443 true xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx https://data.lab.fiware.org/dataset/logrono_cygnus/resource/ca73a799-9c71-4618-806e-7bd0ca1911f4 https://data.lab.fiware.org/dataset/logrono_cygnus 1000
```

Comment:
My eyes just went straight to the xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx :D and it was quite "complex" to check that it referred to the <ckan API key> option... Maybe a comment in the "An example of run:" line changing it to something such as "An example of run (using a fake CKAN API 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'):" may help... ;) No need to say that it is not really important... :)